src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_img2img.py

```diff
@@ -439,7 +421,7 @@ def decode_latents(self, latents):
         image = self.vae.decode(latents).sample
         image = (image / 2 + 0.5).clamp(0, 1)
         # we always cast to float32 as this does not cause significant overhead and is compatible with bfloat16
-        image = image.cpu().permute(0, 2, 3, 1).float().numpy()
+        # image = image.cpu().permute(0, 2, 3, 1).float().numpy()
```

Comment:
```suggestion
        image = image.permute(0, 2, 3, 1)
```

This can stay, only the to CPU etc.. should be removed and done in the postprocessing instead

Reply:
I didn't apply this change here because based on my understanding, torch tensors are always channel-first( i.e. with shape `[B,C,H,W]`), vs numpy arrays are always channel-last (i.e. with shape `[B,H,W,C]`) - this applies for both inputs and output 
So i only permute when we converting a torch tensor into numpy or verse visa